### PR TITLE
doc restructure

### PR DESCRIPTION
**Pull request by melrchen**
there is a vision going on 

image

### CODE REVIEW

**Review by michaelwiktorek — APPROVED** [review]

File: fraud-tutorial.mdx [hunk 1]
```diff
@@ -0,0 +1,951 @@
...
+from accounts;
+```
+
+There are a few differences between this SQL file resolver and the one
+we wrote for the `User` feature class.
+
+First, we've added a `type` key to the header. This tells Chalk
+that this resolver should be used to ingest data into the offline store.
+If we didn't include this key, Chalk would assume that this resolver
+could be queried in realtime.
```
Comment (hugokitano) [inline fraud-tutorial.mdx:477]: link online/offline docs

**Review by hugokitano — APPROVED** [review]
amazing!! thanks for doing this

File: fraud-tutorial.mdx [hunk 2]
```diff
@@ -0,0 +1,951 @@
...
+class User:
+-   id: int
++   user_id: Primary[int]
+    name: str
+    email: str
+```
+
+### Tags, Descriptions, & Owners
+
+In our features below, we've added some comments and
```
[the remark this answers is not in the document]
Comment (melrchen) [reply fraud-tutorial.mdx:124]: LOL

File: fraud-tutorial.mdx [hunk 3]
```diff
@@ -0,0 +1,951 @@
...
++     PENDING = "pending"
++     CLEARED = "cleared"
++     FAILED = "failed"
++
++ @features
++ class Transaction:
++    id: int
++
++    # The id of the account that this transaction belongs to, set to a join.
++    account_id: "Account.id"
```
[the remark this answers is not in the document]
Comment (melrchen) [reply fraud-tutorial.mdx:224]: it is not i add

**Pull request merged**